lms/static/sass/_build-learner-dashboard.scss

```diff
@@ -9,3 +9,4 @@
 @import 'elements/progress-circle';
 @import 'views/program-details';
 @import 'views/program-list';
+@import "views/program-marketing-page";
```

Comment:
Source files are within `sass/course`.
Build file is called `_build-course.scss`.